### PR TITLE
Added screenshots and modified the post date as per issue #213.

### PR DESCRIPTION
**Pull request by anenadic**
Added all the screenshots (partially addresses issue #244 too) and changed the date of the second post. Changed the text a bit too to refer to the date of the second post and also how to view the newly added posts (how to compose the view URL). Fixes issue #213.

Note that the screenshots may look slightly differently than expected as this lesson follows the lesson on layouts which has been modified in a separate PR #242.

### CODE REVIEW

File: _episodes/filters.md [hunk 1]
```diff
@@ -69,31 +69,34 @@ Today was a good day. I was promoted to Surgeon to the Forces!{% endraw %}
 ~~~
 {: .language-markdown }
 
-FIXME: add screenshot of rendered blog post.
+You can view the post at `https://GITHUB_USERNAME.github.io/REPOSITORY_NAME/1827-11-22-surgeon.html`.
+
+![Blog post 1851-05-06](../fig/filters-post-1827-11-22-surgeon.png){: width="700px"}
```
Comment (tobyhodges) [inline _episodes/filters.md]:
```suggestion
![Blog post 1827-11-22](../fig/filters-post-1827-11-22-surgeon.png){: width="700px"}
```

**Review by tobyhodges — COMMENTED** [review]
Just one suggestion, because the date strings didn't match between the alt text and the image filename. Otherwise, looks good to me! I will review #242 later today, when I have time to pull and build the branch locally

**Comment by anenadic** [comment]
> Just one suggestion, because the date strings didn't match between the alt text and the image filename. Otherwise, looks good to me! I will review #242 later today, when I have time to pull and build the branch locally

Thanks! Typical copy + paste mistake :-).

**Comment by tobyhodges** [comment]
Thanks @anenadic !

**Pull request merged**